client/lib/wpcom-undocumented/lib/site.js

```diff
@@ -253,6 +253,18 @@ UndocumentedSite.prototype.getConnection = function( connectionId ) {
 };
 
 /**
+ * Runs Theme Setup (Headstart).
+ *
+ * @return {Promise} A Promise to resolve when complete.
+ */
+UndocumentedSite.prototype.runThemeSetup = function() {
+	return this.wpcom.req.post( {
+		path: `/sites/${ this._id }/theme-setup`,
```

Comment:
The only thing I'm wondering is about this interpolated string, since it's not done anywhere else in this file and there's (from what I can see?) no other ES6 in this file. I imagine everything is transpiled, but maybe there's a case where that's not true?